server/src/main/java/org/elasticsearch/common/settings/ClusterSettings.java

```diff
@@ -181,323 +181,322 @@ public void apply(Settings value, Settings current, Settings previous) {
-                    ResourceWatcherService.ENABLED,
-                    ResourceWatcherService.RELOAD_INTERVAL_HIGH,
-                    ResourceWatcherService.RELOAD_INTERVAL_MEDIUM,
-                    ResourceWatcherService.RELOAD_INTERVAL_LOW,
-                    SearchModule.INDICES_MAX_CLAUSE_COUNT_SETTING,
-                    ThreadPool.ESTIMATED_TIME_INTERVAL_SETTING,
-                    FastVectorHighlighter.SETTING_TV_HIGHLIGHT_MULTI_VALUE,
-                    Node.BREAKER_TYPE_KEY,
-                    OperationRouting.USE_ADAPTIVE_REPLICA_SELECTION_SETTING,
-                    IndexGraveyard.SETTING_MAX_TOMBSTONES,
-                    PersistentTasksClusterService.CLUSTER_TASKS_ALLOCATION_RECHECK_INTERVAL_SETTING,
-                    EnableAssignmentDecider.CLUSTER_TASKS_ALLOCATION_ENABLE_SETTING,
-                    PeerFinder.DISCOVERY_FIND_PEERS_INTERVAL_SETTING,
-                    PeerFinder.DISCOVERY_REQUEST_PEERS_TIMEOUT_SETTING,
-                    ClusterFormationFailureHelper.DISCOVERY_CLUSTER_FORMATION_WARNING_TIMEOUT_SETTING,
-                    ElectionSchedulerFactory.ELECTION_INITIAL_TIMEOUT_SETTING,
-                    ElectionSchedulerFactory.ELECTION_BACK_OFF_TIME_SETTING,
-                    ElectionSchedulerFactory.ELECTION_MAX_TIMEOUT_SETTING,
-                    ElectionSchedulerFactory.ELECTION_DURATION_SETTING,
-                    Coordinator.PUBLISH_TIMEOUT_SETTING,
-                    JoinHelper.JOIN_TIMEOUT_SETTING,
-                    FollowersChecker.FOLLOWER_CHECK_TIMEOUT_SETTING,
-                    FollowersChecker.FOLLOWER_CHECK_INTERVAL_SETTING,
-                    FollowersChecker.FOLLOWER_CHECK_RETRY_COUNT_SETTING,
-                    LeaderChecker.LEADER_CHECK_TIMEOUT_SETTING,
-                    LeaderChecker.LEADER_CHECK_INTERVAL_SETTING,
-                    LeaderChecker.LEADER_CHECK_RETRY_COUNT_SETTING,
-                    Reconfigurator.CLUSTER_AUTO_SHRINK_VOTING_CONFIGURATION,
-                    TransportAddVotingConfigExclusionsAction.MAXIMUM_VOTING_CONFIG_EXCLUSIONS_SETTING,
-                    ClusterBootstrapService.INITIAL_MASTER_NODES_SETTING,
-                    ClusterBootstrapService.UNCONFIGURED_BOOTSTRAP_TIMEOUT_SETTING,
-                    LagDetector.CLUSTER_FOLLOWER_LAG_TIMEOUT_SETTING,
-                    DiscoveryUpgradeService.BWC_PING_TIMEOUT_SETTING,
-                    DiscoveryUpgradeService.ENABLE_UNSAFE_BOOTSTRAPPING_ON_UPGRADE_SETTING
-            )));
+    public static Set<Setting<?>> BUILT_IN_CLUSTER_SETTINGS = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(
+            AwarenessAllocationDecider.CLUSTER_ROUTING_ALLOCATION_AWARENESS_ATTRIBUTE_SETTING,
+            TransportClient.CLIENT_TRANSPORT_NODES_SAMPLER_INTERVAL, // TODO these transport client settings are kind
+            // of odd here and should only be valid if we are a transport client
```

Comment:
Can you wrap this comment onto the preceding line (assuming it fits)? This setting doesn't exist in master anyway.

Reply:
Or, even better, drop the comment altogether. We addressed the TODO in master by dropping the transport client. We aren't going to address this comment in 7.x so it's impotent there.